Explorer/Assets/DCL/FeatureFlags/FeatureFlagOptions.cs

```diff
@@ -19,20 +21,30 @@ public struct FeatureFlagOptions
         /// </summary>
         public string Hostname { get; set; }
 
-        public static FeatureFlagOptions ORG = new()
-        {
-            AppName = "explorer",
-            URL = URLDomain.FromString("https://feature-flags.decentraland.org"),
-            Debug = false,
-            Hostname = "https://decentraland.org",
-        };
+        private static FeatureFlagOptions NewOrg() =>
+            new ()
+            {
+                AppName = "explorer",
+                URL = URLDomain.FromString("https://feature-flags.decentraland.org"),
+                Debug = false,
+                Hostname = "https://decentraland.org",
+            };
 
-        public static FeatureFlagOptions ZONE = new()
-        {
-            AppName = "explorer",
-            URL = URLDomain.FromString("https://feature-flags.decentraland.zone"),
-            Debug = false,
-            Hostname = "https://decentraland.zone",
-        };
+        private static FeatureFlagOptions NewZone() =>
+            new ()
+            {
+                AppName = "explorer",
+                URL = URLDomain.FromString("https://feature-flags.decentraland.zone"),
+                Debug = false,
+                Hostname = "https://decentraland.zone",
```

Comment:
`Hostname` should be resolved from `DecentralandUrl.Host` so we avoid a duplicate